README.md

```diff
@@ -1,6 +1,201 @@
-# KubernetesAccessManager
-KubernetesAccessManager is a web application designed for multi-user management of Kubernetes cluster. It provides a graphical interface for users to manage their Kubernetes resources. Administrators can define precise access rights based on roles from an identity provider, such as Keycloak.
+<h1 style="display: flex; align-items: center; gap: 30px;">
+  Kubernetes Access Manager (KAM)
+  <img src="./docs/images/KAM_logo.png" alt="KAM Logo" style="height: 125px;">
+</h1>
 
-## Project Status
+---
+## For English Press [Here](#documentation-in-english) / Для русского нажмите [здесь](#документация-на-русском-языке)
 
-**Note:** This project is currently under development and is not ready for production use. We don't accept any contributions at this time.
+---
+## Dokumentacja w języku polskim
+
+Aplikacja **Kubernetes Access Manager (KAM)** pomaga rozwiązać problem braku elastyczności w zarządzaniu dostępem do 
+klastrów Kubernetes. Dzięki precyzyjnemu przypisywaniu uprawnień na podstawie przestrzeni nazw (namespace) i typów 
+zasobów (resource), KAM upraszcza zarządzanie klastrami i zwiększa bezpieczeństwo.
+
+Zintegrowana z systemami zarządzania tożsamością, takimi jak Keycloak, aplikacja wykorzystuje standard OpenID Connect 
+(OIDC), zapewniając łatwą autoryzację i uwierzytelnianie.
+
+---
+
+## Najważniejsze funkcje
+
+- **Zarządzanie zasobami Kubernetes**: Tworzenie, edycja, usuwanie.
+- **Przegląd zasobów**:
+    - Wyświetlanie listy zasobów.
+    - Szczegółowe informacje o zasobach i aplikacjach helmowych.
+- **Rollback aplikacji helmowych**: Przywracanie wcześniejszych wersji.
+- **Zarządzanie dostępem**:
+    - 5 poziomów uprawnień:
+        - Widok listy.
+        - Widok szczegółowy.
+        - Tworzenie.
+        - Edycja.
+        - Usuwanie.
```

Comment:
Powtarzasz się tu

Reply:
No niby jo, ale jedno się dotyczy dostępów a drugie tego co można robić na klastre ogólnie - nie za bardzo wiem jakby to inaczej opisać żeby się niepowtarzać, a był ten podział